SU-HSDO/suhumsci
# Stn 945  vertical timeline arrow toggle animation

# READY FOR REVIEW

## Summary
The desired effect for each timeline item is that when the summary is clicked, the content expands, the arrow animates to point down, and the aria-expanded and aria-pressed attributes change to "true". The undesired effect that was occurring was that when the entire timeline item was pressed (such as in the expanded content area), the aria attributes were triggered as false and the arrow animated to point up. This should not occur unless the summary area only is clicked.

## Need Review By (Date)
asap

## Urgency
medium

## Steps to Test
1. Run npm test
2. Import expand-collapsse-timeline.js
3. Create a page with a timeline item. 
4. Click the item to expand it. The arrow should animate and aria attributes change to "true".
5. Click on the expanded content to ensure the aria attributes and arrow animation are not effected.
6. Click on the summary area which was originally clicked to cause the expand effect. Check that the content collapses, arrow animates, and aria attributes change to "false".

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Sparkbox PR Checklist](../docs/SparkboxPRChecklist.md)
